bazel/python_repositories.bzl

```diff
@@ -21,3 +21,25 @@ def python_repositories():
         shallow_since = "1548652954 +0800",
         remote = "https://github.com/URenko/aioh2.git",
     )
+    new_git_repository(
+        name = "ryu_repo",
+        build_file = "//bazel/external:ryu.BUILD",
+        commit = "c776e4cb68600b2ee0a4f38364f4a355502777f1",  # Corresponds to: tag = "v4.34"
+        shallow_since = "1569926530 +0900",
+        remote = "https://github.com/faucetsdn/ryu.git",
```

Comment:
I'm wondering how we built this before. If I search for "faucetsdn" in the code, I find nothing so far. If I search for "deb-python-aioeventlet" (from the block below), I find a file `aioeventlet_build.sh`

Reply:
I see. And yes, I think it's possibly applied in two places, the `py_patches` target even has some explicit logic for dealing with already applied patches. Anyway, the new way is so much cleaner than modifying the already installed files!